micro_manager/micro_manager.py

```diff
@@ -654,6 +755,110 @@ def _solve_micro_simulations_with_adaptivity(
 
         return micro_sims_output
 
+    def _interpolate_output_for_crashed_sim(
+        self,
+        micro_sims_input: list,
+        micro_sims_output: list,
+        unset_sim: int,
+        active_sim_ids: np.ndarray = None,
+    ) -> dict:
+        """
+        Using the output of neighboring simulations, interpolate the output for a crashed simulation.
+
+        Parameters
+        ----------
+        micro_sims_input : list
+            List of dicts in which keys are names of data and the values are the data which are required inputs to
+            solve a micro simulation.
+        micro_sims_output : list
+            List dicts containing output of local micro simulations.
+        unset_sim : int
+            Index of the crashed simulation in the list of all local simulations currently interpolating.
+        active_sim_ids : numpy.ndarray, optional
+            Array of active simulation IDs.
+
+        Returns
+        -------
+        output_interpol : dict
+            Result of the interpolation in which keys are names of data and the values are the data.
+        """
+        # Find neighbors of the crashed simulation in active and non-crashed simulations
+        # Set iteration length to only iterate over active simulations
+        if self._is_adaptivity_on:
+            iter_length = active_sim_ids
+        else:
+            iter_length = range(len(micro_sims_input))
+        micro_sims_active_input_lists = []  # DECLARATION
+        micro_sims_active_values = []  # DECLARATION
```

Comment:
```suggestion
        micro_sims_active_input_lists = []
        micro_sims_active_values = []
```
The `DECLARATION` tag is used for member variables and not local variables within a function.